src/Avalonia.Controls/DataValidationErrors.cs

```diff
@@ -5,7 +5,8 @@
 using Avalonia.Controls.Metadata;
 using Avalonia.Controls.Templates;
 using Avalonia.Data;
-
+using Avalonia.Automation.Peers;
+using Avalonia.Controls.Automation.Peers;
```

Comment:
Newline is in the wrong spot. You shouldn't have usings right next to the class with no space.